Cargo.toml

```diff
@@ -1,20 +1,21 @@
 [package]
 edition = "2021"
 name = "bzip2"
-version = "0.4.4"
+version = "0.5.0"
 authors = ["Alex Crichton <alex@alexcrichton.com>"]
```

Comment:
A minor thing, but using the authors field is no longer encouraged and both crates.io and docs.rs no longer show it anywhere: https://rust-lang.github.io/rfcs/3052-optional-authors-field.html Maybe ask Alex if he wants to keep this field around or not?

Reply:
I think we can just get rid of it then; github keeps track of the score